src/js/controller/events-middleware.ts

```diff
@@ -1,6 +1,7 @@
-import { GenericObject, CoreModel } from 'types/generic.type';
+import { GenericObject } from 'types/generic.type';
+import Model from './model';
```

Comment:
did we see if this gets stripped out even without `import type` - little unclear on your previous comment

Reply:
It doesn't matter for modules used in the controller. What we want to avoid is importing model into modules imported in the embed.